app/assets/stylesheets/task.css

```diff
@@ -0,0 +1,66 @@
+table, th {
+  padding: 10px;
+  margin-left: auto;
+  margin-right: auto;
+  background-color: #FFFFCC;
+  text-align: left;
+}
+
+th {
+  text-decoration: underline;
+  text-align: center;
+}
+
+td {
+  text-align: center;
+}
+
+h1 {
+  color: #FFFFFF;
+}
+
+body {
+  background-image: image-url("puppy.jpg");
```

Comment:
This is odd that this doesn't work because the `image-url` thing worked for other folks so I'm not sure what's wrong here